lib/chef/exceptions.rb

```diff
@@ -259,7 +259,7 @@ class InvalidSymlink < RuntimeError; end
     class ChildConvergeError < RuntimeError; end
 
     class DeprecatedFeatureError < RuntimeError
-      def initalize(message)
+      def initialize(message)
```

Comment:
Any specific reason to put `initalize` instead of `initialize`?

If No, let's make it happy!!!

Reply:
no, that's a straight up bug